admin/scss/bootstrap/_variables.scss

```diff
@@ -57,7 +57,7 @@ $brand-danger:              #D40404;
 // Control the default styling of most Bootstrap elements by modifying these
 // variables. Mostly focused on spacing.
 
-$spacer:                     1rem;
+$spacer:                     1.2308rem;
```

Comment:
This convention is taken from bootstrap: https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L164. Since devs wouldn't be expected to calculate or copy these values around themselves, its a low priority to figure out. Paul said something about independence between font-sizes and spacers, but they're related through line-height. Getting a bit too complicated for me, so I'm leaving it as-is ;)